packages/chain-events/services/ChainEventsConsumer/ChainEventHandlers/storage.ts

```diff
@@ -58,6 +61,16 @@ export default class extends IEventHandler {
   }
 
   private async _shouldSkip(event: CWEvent): Promise<boolean> {
+    // filter out all events that won't have an associated entity in the db
+    const [kind, type] = eventToEntity(event.network, event.data.kind);
```

Comment:
add entity id is null or not check

Reply:
Yep -- this should be a null check rather than checking against `type`. Current code will throw error on destructure.

Reply:
added this - little refactoring - untested though!